packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/v6/CodeMirror.tsx

```
@@ -66,7 +66,7 @@ const CodeMirror = (props: NoteBodyEditorProps, ref: ForwardedRef<NoteBodyEditor
 	const rootRef = useRef(null);
 	const webviewRef = useRef(null);
 
-	type OnChangeCallback = (event: OnChangeEvent)=> void;
+	type OnChangeCallback = (event: OnChangeEvent) => void;
```

ESLint

[error] 'event' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)